test/aws/osml/models/flood/test_flood_model.py

```diff
@@ -59,4 +58,4 @@ def test_predict_bad_data_file(self):
         data_binary = None
         response = self.client.post("/invocations", data=data_binary)
 
-        assert response.status_code == 400
+        assert response.status_code == 500
```

Comment:
Same comment on centerpoint model. Seems like 4xx is correct.

Reply:
Will update